Mods/EpipEncounters_7d32cb52-1cfd-4526-9b84-db4867bf9356/Story/RawFiles/Lua/Epip/QuickInventory/Client.lua

```diff
@@ -100,6 +100,11 @@ local QuickInventory = {
            Text = "Quick Swap...",
            ContextDescription = "Context menu button label",
         },
+        Setting_Sorting_Name = {
```

Comment:
This word is generic enough that it could be added to CommonStrings instead, so that in case anything else uses it in the future it needn't be translated twice